emails/views.py

```diff
@@ -65,6 +65,23 @@ def __init__(self, message="No In-Reply-To header."):
         self.message = message
 
 
+def wrapped_email_test(request):
+    html_content = '<p><strong>strong</strong></p><hr><p>plain</p>'
+    display_email = 'test@relay.firefox.com'
+    attachments = None
+    user_profile = Profile.objects.order_by('?').first()
```

Comment:
what does `order_by('?')` do?

Reply:
It means "order randomly" - https://docs.djangoproject.com/en/4.0/ref/models/querysets/#order-by